src/1-Utilities/utilities.ts

```diff
@@ -21,3 +21,21 @@ export const circleFilterUnique = (list:CircleListItem[]):CircleListItem[] =>
 
 //Converts underscores to spaces and capitalizes each word
 export const makeDisplayText = (text:string = ''):string => text.toLowerCase().split('_'||' ').map((s) => s.charAt(0).toUpperCase() + s.substring(1)).join(' ');
+
+
+/* Transform inputMap to Simple JavaScript Object */
+export const assembleRequestBody = (inputMap:Map<string,any>):Object => {
```

Comment:
On the portal, when I try and sign up a new user, i'm getting a ReferenceError stating that finalMap is not defined.